BuildForSDG/Team-196-Project
# Ft user specialist mapping

## Description
1. Added SpecialistMapper model, associations, and validations.
2. Added SpecialistMapper controller.
3. Added unit and integration tests.

Fixes #5 

## How Has This Been Tested?
1. Clone the repository.
2. Check out to `ft-user-specialist-mappings` branch.
3. Run `bundle install`.
4. To set up the database, run `rails db: setup`
4. Run `rails db:migrate`  to create the database schema.
5. Enter `bundle exec rspec` to run the tests.


## Checklist:

****
